engine/src/main/java/org/pentaho/di/job/entry/JobEntryBase.java

```diff
@@ -1520,4 +1524,28 @@ public JobMeta getParentJobMeta() {
   public void setParentJobMeta( JobMeta parentJobMeta ) {
     this.parentJobMeta = parentJobMeta;
   }
+
+  /**
+   * Gets a Map of variables set in EntryStepSetVariables
+   *
+   * @return a map of variable names and values
+   */
+  public Map<String, String> getEntryStepSetVariablesMap() {
```

Comment:
We should not expose the map outside the class. Please make this protected instead of public.